packages/nova-react/src/graphql/hooks.test.tsx

```diff
@@ -126,6 +132,126 @@ describe(useFragment, () => {
   });
 });
 
+describe(useRefetchableFragment, () => {
+  it("uses the host's hook, if provided", () => {
+    const graphql: NovaGraphQL = {
+      useRefetchableFragment: jest.fn(() => [
+        { someKey: "some-data" },
+        ({}) => ({ dispose: jest.fn() }),
+      ]),
+    };
+
+    const fragment = {} as unknown as GraphQLTaggedNode;
+    const fragmentRef = {} as any;
+
+    const Subject: React.FC = () => {
+      const [data, _] = useRefetchableFragment(fragment, fragmentRef);
+      return <span>{data.someKey}</span>;
+    };
+
+    render(
+      <NovaGraphQLProvider graphql={graphql}>
+        <Subject />
+      </NovaGraphQLProvider>,
+    );
+
+    expect(graphql.useRefetchableFragment).toHaveBeenCalledWith(
+      fragment,
+      fragmentRef,
+    );
+    expect(screen.getByText("some-data")).toBeDefined();
+  });
+
+  it("ensures an implementation is supplied", () => {
+    const graphql: NovaGraphQL = {
+      useRefetchableFragment: undefined,
+    };
+
+    const fragment = {} as unknown as GraphQLTaggedNode;
+    const fragmentRef = { someKey: "some-data" } as any;
+
+    const Subject: React.FC = () => {
+      const [data, _] = useRefetchableFragment(fragment, fragmentRef);
+      return <span>{data.someKey}</span>;
+    };
+
+    expect(() =>
+      render(
+        <NovaGraphQLProvider graphql={graphql}>
+          <Subject />
+        </NovaGraphQLProvider>,
+      ),
+    ).toThrowErrorMatchingInlineSnapshot(
+      `"Expected host to provide a useRefetchableFragment hook"`,
+    );
+  });
+});
+
+describe(usePaginationFragment, () => {
+  it("uses the host's hook, if provided", () => {
+    const mockedResponse = {
+      data: {
+        someKey: "some-data",
+      },
+      loadNext: jest.fn(),
+      loadPrevious: jest.fn(),
+      hasNext: false,
+      hasPrevious: false,
+      isLoadingNext: false,
+      isLoadingPrevious: false,
+      refetch: jest.fn(),
+    };
+
+    const graphql: NovaGraphQL = {
+      usePaginationFragment: jest.fn(() => mockedResponse),
+    };
+
+    const fragment = {} as unknown as GraphQLTaggedNode;
```

Comment:
Instead of `unknown` as `GraphQlTaggedNode` , use this -> `{ __brand: "GraphQLTaggedNode" }`